content/WebAuthn/Concepts/Using_WebAuthn_for_Signing.adoc

```diff
@@ -5,21 +5,23 @@ While WebAuthn was designed primarily for authentication, the framework can be e
 
 With the MakeCredential operation creating a keypair with the algorithm defined, the link:https://www.w3.org/TR/webauthn-2/#sctn-op-get-assertion[GetAssertion] operation is used to actually sign the data. The GetAssertion flow also supplies a challenge from the relying party to the authenticator to be signed as part of the link:https://www.w3.org/TR/webauthn-2/#sctn-verifying-assertion[Authentication] ceremony. This challenge can be replaced with a hash of the document to be signed, creating an association with the document or data set to be signed. The method in which this hash is generated should be determined by the capabilities of the environment and security requirements under which the application is run. 
 
-No modifications are required to this operation to support the signature, but both the link:https://www.w3.org/TR/webauthn-1/#dictdef-collectedclientdata[client data] and link:https://www.w3.org/TR/webauthn-1/#sec-authenticator-data[authenticator data] will need to be recorded along with the returned signature. This is because the signature returned by WebAuthn is for an object comprising both the client data (including the hash string) and authenticator data, and both sets of data will be required to validate the signature in the future. To assist with the, the PublicKeyCredential returned will include the client data (as a JSON object), the authenticator data and the signature. Simply archiving everything returned will allow for ease of signature validation. In a pinch, the client data does not absolutely need to be retained as it can be reconstructed as long as the domain is known and the relying party remains operational, but for ease of access and export, there are benefits for simply storing it. Other metadata, such as the timestamp of the signing event, may be useful to include as well.
+No modifications are required to this operation to support the signature, but both the link:https://www.w3.org/TR/webauthn-1/#dictdef-collectedclientdata[client data] and link:https://www.w3.org/TR/webauthn-1/#sec-authenticator-data[authenticator data] will need to be recorded along with the returned signature. This is because the signature returned by WebAuthn is for an object comprising both the client data (including the hash string) and authenticator data, and both sets of data will be required to validate the signature in the future. To assist with the, the PublicKeyCredential returned will include the client data (as a JSON object), the authenticator data and the signature. Simply archiving everything returned will allow for ease of signature validation. Other metadata, such as the timestamp of the signing event, may be useful to include as well.
```

Comment:
Typo: "To assist with the, the PublicKeyCredential". Should be "To assist with this, the PublicKeyCredential"?